process-engine/process_engine
# Fix overwritten LaneSet Property in the Process Model Persistence Service

## What did you change?
This PR fixes a bug, where the LaneSet Property of the passed ProcessModel where accidentaly overwritten.  This creates an issue, where it was not possible to execute any process, after a process failed with a 403 Error. 

In this PR the ProcessModel object gets cloned before the LaneSet Property of the process model will be overwritten in the method `_filterInaccessibleProcessModelElements`, called in `getProcessModelById`.

## How can others test the changes?
* Run the Integration Tests of the ConsumerAPI
* See, if the test order matters

## PR-Checklist

Please check the boxes in this list after submitting your PR:
